coriolis/exception.py

```diff
@@ -470,3 +468,27 @@ class FailedPackageUninstallationException(PackageManagerOperationException):
     message = (
         "Failed to remove unwanted packages (%(package_names)s) through "
         "%(package_manager)s. Error was: %(error)s")
+
+
+class OSMorphingOperationTimeout(OSMorphingException):
+    pass
+
+
+class OSMorphingSSHOperationTimeout(OSMorphingOperationTimeout):
+    message = (
+        "Pending SSH command %(cmd)s timed out after %(timeout)s seconds. "
+        "Coriolis may have encountered connection issues to the minion machine"
+        " or the command execution time exceeds the timeout set. Try extending"
+        " the timeout by editing the 'default_osmorphing_operation_timeouts' "
+        "in /etc/coriolis/coriolis.conf."
```

Comment:
NIT: please no explicit paths like `/etc/coriolis/coriolis.conf` since those are not necessarily set in stone.

Please use "Coriolis' static configuration file" or similar.